src/test/java/org/springframework/data/tarantool/repository/support/CompositePkIntegrationTest.java

```diff
@@ -0,0 +1,151 @@
+package org.springframework.data.tarantool.repository.support;
+
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.data.tarantool.BaseIntegrationTest;
+import org.springframework.data.tarantool.BaseIntegrationTestStub;
+import org.springframework.data.tarantool.entities.BookTranslation;
+import org.springframework.data.tarantool.entities.BookTranslationId;
+import org.springframework.data.tarantool.repository.BookTranslationRepository;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import java.util.Optional;
+
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.junit.jupiter.api.Assertions.assertTrue;
+
+/**
+ * @author Alexey Kuzin
+ */
+@Tag("integration")
+class CompositePkIntegrationTest extends BaseIntegrationTest {
```

Comment:
Add test for checking the error if there is no `TarantoolIdClass`but  there are several properties with `@Id` annotation

Reply:
Checked in BookStoreId